store/store.md

```diff
@@ -1 +1,60 @@
 # Store
+
+## Abstract
+
+The Store interface defines methods for storing and retrieving blocks, commits, and the state of the blockchain.
+
+## Protocol/Component Description
+
+The Store interface defines the following methods:
+
+- `Height`: Returns the height of the highest block in the store.
+- `SetHeight`: Sets the height saved in the Store if it is higher than the existing height.
+- `SaveBlock`: Saves a block along with its seen commit.
+- `LoadBlock`: Returns a block at a given height.
+- `LoadBlockByHash`: Returns a block with a given block header hash.
+- `SaveBlockResponses`: Saves block responses in the Store.
+- `LoadBlockResponses`: Returns block results at a given height.
+- `LoadCommit`: Returns a commit for a block at a given height.
+- `LoadCommitByHash`: Returns a commit for a block with a given block header hash.
```

Comment:
Are all the load methods loading into memory? Can we specify?

Reply:
All the load methods are just get methods that return the value fetched from the underlying database. Perhaps, we should use the word `Get` instead of `Load` here?

Reply:
Yea let's make a follow up to rename to `Get`

Reply:
Follow up issue: https://github.com/rollkit/rollkit/issues/1284